src/main/java/seedu/duke/Parser.java

```diff
@@ -0,0 +1,70 @@
+package seedu.duke;
+
+import seedu.exceptions.DukeException;
+import seedu.exceptions.InvalidTaskNameException;
+import seedu.task.TaskList;
+
+import java.util.Scanner;
+
+/**
+ *  This UI class contains methods that print out chatbot
+ *  messages in the console to the user.
+ */
+public class Parser extends Ui {
+
+    public Parser(TaskList tasks) {
+        super(tasks);
+    }
+
+    public void parse() {
```

Comment:
This method is public, and hence should be commented with javadocs, according tot he module's coding conventions